src/config/dynamic-theme-fixes.config

```diff
@@ -14767,6 +14767,15 @@ body,
 .searchTitle {
     background: none !important;
 }
+.sc-classic .header__navMenu > li > a {
+    border-right-color: #242424 !important;
+}
+.headerSearch__input {
+    background: #1e2122 !important;
+}
+.g-dark a {
+   color: #cccccc;
+}
```

Comment:
I don't think that little of a difference is consider a fix.

Reply:
oh that, the g-dark line, yeah, I added that to restore the header button hovers, because without it, you wouldn't be able to see anything when you hover over it while in soundcloud, normally, their header buttons have hovers, so I want to kind of, bring that back you know?

Reply:
>  I added that to restore the header button hovers while in soundcloud

From what I see with default settings that is already happening? Do you happen to have a custom configuration?

Reply:
> > I added that to restore the header button hovers while in soundcloud
> 
> From what I see with default settings that is already happening? Do you happen to have a custom configuration?

oh you're right, it does, but it's very unnoticable, so I decided to make it more noticable that a user is hovering on it & no, I do not have a custom config.